R/resolve_secrets.R

```diff
@@ -0,0 +1,46 @@
+#' Use vault to resolve secrets.
+#' 
+#' Sets up a vault client via call to \code{\link{vault_client}} and resolves
+#' secrets specified by the form VAULT:<path to secret>:<key to get>.
+#'
+#' @param x List of properties values of which may be vault secrets of the form
+#' VAULT:<path to secret>:<key to get>. Any values which don't match pattern
+#' of a secret are ignored and returned as is.
+#' @param ... Args to be passed to \code{\link{vault_client}} call.
+#' @param login Login method to be passed to call to \code{\link{vault_client}}.
+#'
+#' @return List of properties with any vault secrets resolved.
+#' @export
+#' 
+#' @examples
+#' \dontrun{
```

Comment:
I can remove that, just thought it would be the easiest way to show how we are expecting it to be used. As in setting the environmental variables for authenticating against the vault.

Reply:
please leave it in for now - it might be possible to make it a bit clearer is all